exercises/exercise3.js

```diff
@@ -1,12 +1,444 @@
+        return (
+          "Nice job " +
+          state.firstName +
+          ", " +
+          `${
+            answer13.charAt(0).toUpperCase() + answer13.slice(1).toLowerCase()
+          }` +
+          " is the correct answer. Well done! Your score is " + (state.score += 10) + "." + "\nNext question: Kuala Lumpur is the capital of which country?"
+        );
+      } else {
+        return "Sorry, that's wrong. To call you stupid would be an insult to stupid people! I've known sheep that could outwit you.";
+      }
+    },
+    isListening: true,
+  },
+  {
+    label: "Question 14",
+    prompt: "Q14: #{answer14}",
+    handler: function (params, state) {
+      const answer14 = params.answer14;
+      if (answer14.toLowerCase() === "malaysia") {
+        return (
+          "Nice job " +
+          state.firstName +
+          ", " +
+          answer14 +
+          " is the correct answer. Well done! Your score is " + (state.score += 10) + "." + "\nNext question: Which country is called the Land of Rising Sun?"
+        );
+      } else {
+        return "Sorry, that's wrong. Your brain's so minute that if a hungry cannibal cracked your head open, there wouldn't be enough to cover a small water biscuit.";
+      }
+    },
+    isListening: true,
+  },
+  {
+    label: "Question 15",
+    prompt: "Q15: #{answer15}",
+    handler: function (params, state) {
+      const answer15 = params.answer15;
+      if (answer15.toLowerCase() === "japan") {
+        return (
+          "Nice job " +
+          state.firstName +
+          ", " +
+          `${
+            answer15.charAt(0).toUpperCase() + answer15.slice(1).toLowerCase()
+          }` +
+          " is the correct answer. Well done! Your score is " + (state.score += 10) + "." + "\nThe next question is the final question, it's hard.\nNext question: What is the capital city of Slovenia, formerly part of the state of Yugoslavia?"
+        );
+      } else {
+        return "Sorry, that's wrong. You are a fart factory, slug-slimed sack of rat guts in cat vomit. A cheesy scab picked pimple squeezing finger bandage. \nA week old maggot burger with everything on it and flies on the side.";
+      }
+    },
+    isListening: true,
+  },
+  {
+    label: "Final Answer",
+    prompt: "Q16: #{answer16}",
+    handler: function (params, state) {
+      const answer16 = params.answer16;
+      if (answer16.toLowerCase() === "ljubljana") {
+        return (
+          "Nice job " +
+          state.firstName +
+          ", " +
+          `${
+            answer16.charAt(0).toUpperCase() + answer16.slice(1).toLowerCase()
+          }` +
+          " is the correct answer. Well done! Your score is " + (state.score += 10) + "." + "\nYour Geography skills are on point. Go off and teach this stuff instead of trying to punch code."
+        );
+      } else {
+        return "Sorry, that's wrong. You are about one bit short of a byte.";
+      }
+    },
+    isListening: true,
   },
 ];
```

Comment:
How did you come up with so many inventive insults?! I think I stopped playing halfway through just so I could read through them all - haha! Above and beyond Kevin - fantastic job.

Reply:
I Googled "20 best insults" and then just copy paste! :)